app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted = command.split()
+
+    if splitted[0] == "cp" and splitted[1] != splitted[2]:
+        file1 = splitted[1]
+        file2 = splitted[2]
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
command, old_file, new_file = command.split()